website/src/data/cloud-images.json

```diff
@@ -489,250 +489,290 @@
   },
   {
     "cloud": "aws",
-    "version": "v0.9.0",
-    "region": "ca-central-1",
+    "version": "v0.9.3",
```

Comment:
Not for this PR, but was thinking that we should put the AMI list into a collapsable section in the docs.

Reply:
`<details></details>` :+1:

Reply:
That's the one 🙂